src/user/dashboard/settings/layouts/SettingSidebar.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { ListGroup } from "react-bootstrap";
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import {
+  MdSecurity,
+  MdLocationOn,
+  MdNotifications,
+  MdRssFeed,
+  MdBlock,
+  MdAddToQueue,
+  MdLanguage
+} from "react-icons/md";
+import { FaUserTag } from "react-icons/fa"; 
+import "../../../dashboard/navigation/navigation.scss";
+
+class SettingSidebar extends Component {
+  render() {
+    const iconsSize = 40;
+    const { privacy, notification, location, posts, blocked, tagged } = this.props;
+    return (
+     <div className="navigation">
+        <ListGroup>
+          <ListGroup.Item className={privacy ? "active mt-4" : "inactive mt-4"} >
+            <NavLink to="/privacy" className="link">
+              <b><MdSecurity size={iconsSize}/>Security and Privacy</b>
+            </NavLink>
+          </ListGroup.Item>
+          <ListGroup.Item className={location ? "active" : "inactive"}>
+            <NavLink to="/location" className="link">
+              <b><MdLocationOn size={iconsSize}/>Location</b>
+            </NavLink>
+          </ListGroup.Item>
+          <ListGroup.Item
+            className={notification ? "active" : "inactive"}
+          >
```

Comment:
for loop

Reply:
@vaibhavdaren I have already done this part in material-ui branch, and also this PR is on to master branch, so what to do?